lib/beaker/host.rb

```diff
@@ -438,7 +438,7 @@ def do_scp_to source, target_path, options
           @logger.trace result.stdout
         end
       else # a directory with ignores
-        dir_source = Dir.glob("#{source}/**/*").reject do |f|
+        dir_source = Dir.glob("#{source}/**/{*,.*}").reject do |f|
```

Comment:
Was away for a while, just now catching up.
@kevpl Is there a simple example I can dig into to hide this behavior behind an option?